blockchain/contracts/HelloWorld.sol

```diff
@@ -66,16 +90,20 @@ contract HelloWorld {
             nftOwnerships[availableNFTs[0]].dropTime <= block.timestamp,
             "Droptime not yet reached!"
         );
-        uint256 i;
         uint256 nftElement;
         uint256 nftHash;
-        for (i = 0; i < joinedUsers.size(); i++) {
-            nftElement = generateRandomNumber(i) % availableNFTs.length;
-            nftHash = availableNFTs[nftElement];
-            remove(nftElement);
-            nftOwnerships[nftHash].reservedFor = joinedUsers.getAddressAtIndex(
-                i
-            );
+        for (uint256 i = 0; i < joinedUsers.size(); i++) {
```

Comment:
Is there a particular reason to use `uint256` instead of `uint`? Is it faster? Best Practice? The number of people in the world will never fill 64-bit, so 192 bits will always be just zero.

Reply:
uint is unint256...

Reply:
And it would be nice if you could leave out unnecessary comments that have absolutely nothing to do with this pr, like the number of observable atoms in our universe :)

Reply:
Just saying, about 7/8 of the bitlength likely will never be used, probably even more. I don't know how efficiently large numbers are executed in blockchain nodes. But if there is more efficient data type in terms of computational effort and performance I think it should be prefered.